linker/shared/src/main/scala/org/scalajs/linker/frontend/optimizer/IncOptimizer.scala

```diff
@@ -703,6 +777,9 @@ final class IncOptimizer private[optimizer] (config: CommonPhaseConfig, collOps:
       }
     }
 
+    private val jsMethodContainer = new JSMethodContainer(className, this)
+    jsMethodContainer.updateWith(linkedClass)
```

Comment:
Putting the `updateWith` call here is not consistent with `MethodContainer`, for which we put it in the constructor of `MethodContainer`. This confused me for a moment. Could we reconcile them?